portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,8 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import java.util.*;
```

Comment:
usually it's best to just import the classes that you're using, not "*" :) In this case, I think it would be
`import java.util.ArrayList;
import java.util.Arrays;`